tap_pipedrive/streams/filters.py

```diff
@@ -7,4 +7,4 @@ class FiltersStream(PipedriveStream):
     key_properties = [
         "id",
     ]
-    state_field = "add_time"
+    state_field = "update_time"
```

Comment:
As I understood it, the `update_time` does not have values (potentially until they get updated). My previous comment was to fall back to `add_time` if the `update_time` does not have a value to guarantee that you always have a value.

Reply:
It turns out if update_time is None, the state will be the start_date from config file. So we do not need to change update_time to add_time